### ISSUE TITLE
highlight name clicked on

### ISSUE THREAD

**Issue by MalcolmMielle** [cppit/jucipp#364]
When clicking on a variable (or other), the name gets bold and change size thus causing the rest of the line in the document to move. Visually, this makes it really hard to follow what is going on. Any way to make it go bold without moving the rest of the line ?

**Comment by eidheim** [comment]
This should be avoided if you use a monospace font

**Comment by MalcolmMielle** [comment]
Apparemtnly it's [a known problem of Fira](https://github.com/tonsky/FiraCode/issues/552). Any way to change the behavior in Juci++ ?
